requirements.txt

```
@@ -3,3 +3,5 @@ pandas
 numpy
 scikit_learn >= 1.0.0
 julia >= 0.6.0
+click>=8.0.4
+setuptools>=66.0.0
```

```suggestion
setuptools>=50.0.0
```
(Not sure if the constraint is needed)